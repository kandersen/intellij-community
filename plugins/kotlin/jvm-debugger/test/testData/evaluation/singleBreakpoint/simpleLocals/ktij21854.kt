// FILE: common.kt
package ktij21854

fun test() {
    val x = 1
    fun bar(): Int {
        val y = x
        return y
    }
    val c = C()

    val a = 0
}

class C {
    val local: String = ""
}

// FILE: test.kt
package ktij21854
fun main() {
    //Breakpoint!
    test()
}

// STEP_INTO: 1
// STEP_OVER: 2

// EXPRESSION: c.local
// RESULT: "": Ljava/lang/String;
