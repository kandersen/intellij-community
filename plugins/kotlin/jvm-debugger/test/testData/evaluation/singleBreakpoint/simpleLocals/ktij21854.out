LineBreakpoint created at test.kt:5
Run Java
Connected to the target VM
test.kt:5
common.kt:5
common.kt:10
common.kt:12
Compile bytecode for c.local
Disconnected from the target VM

Process finished with exit code 0
